frontend/relay-workflows-lib/lib/components/SubmittedMessagesList.tsx

```diff
@@ -103,14 +157,25 @@ const SubmittedMessagesList: React.FC<SubmittedMessagesListProps> = ({
       }}
     >
       <Divider sx={{ mt: 5, mb: 5 }} />
-      {submissionResults.length > 0 && (
+      {submittedData.length > 0 && (
         <>
           <Paper elevation={1} sx={{ p: 2 }}>
             <Typography variant="h6" sx={{ textAlign: "center" }}>
               Submissions
             </Typography>
           </Paper>
-          {submissionResults.map((r, i) => renderSubmittedMessage(r, i))}
+          {submittedData.map((data, i) =>
```

Comment:
Can this be a set width so that all of the status icons are vertically aligned with one another, including when one of the submissions has an error?